### ISSUE TITLE
Specify tab size for `@stylistic/indentation`, which affects `@stylistic/max-line-length`

### ISSUE THREAD

**Issue by AlexWayfer** [stylelint-stylistic/stylelint-stylistic#10]
### What is the problem you're trying to solve?

Hello.

I know it's controversial, but may be implemented.

I have `indentation: tab` and `max-line-length: 100`.

But the problem is that `max-line-length` counts `tab` as a single character. What, technically, can be right, but causes divergence between code editor and linter output:

![image](https://github.com/firefoxic/stylelint-codeguide/assets/6510020/1c9e8ca7-60ac-45bb-a6a5-b1be72f9be2e)

```css
				--background-color: color-mix(in srgb, var(--button-background-color), var(--link-color) 20%);
```

```
> stylelint styles/

```

If I change `max-line-length` to `97`:

```
styles/index.css
 307:98  ✖  Expected line length to be no more than 97 characters  max-line-length

1 problem (1 error, 0 warnings)
```

### What solution would you like to see?

A new option for `codeguide/indentation` rule. Something like `tabWidth`, which works only when `indentation: tab`.

I know: tabs are meaning to be flexible from developer to developer, etc., there are many questions to such hard-set tab size, but it's perfect for me for now (for years already, actually).

Example: RuboCop supports such thing.

https://docs.rubocop.org/rubocop/cops_layout.html#layoutindentationstyle
https://docs.rubocop.org/rubocop/cops_layout.html#layoutindentationwidth

```yaml
Layout/IndentationStyle:
  EnforcedStyle: tabs
  IndentationWidth: 2
Layout/IndentationWidth:
  Width: 1
```

**Comment by AlexWayfer** [comment]
Oh, and more close example to front-end, from ESLint:

```yaml
rules:
  indent:
    - error
    - tab
    - SwitchCase: 1
  no-mixed-spaces-and-tabs:
    - error
    - smart-tabs
  max-len:
    - error
    - code: 100
      tabWidth: 2
      ignoreUrls: true
```

**Comment by firefoxic** [comment]
Hi, @AlexWayfer! Thanks for your request!
The `tabSize` option will be available in the next release.
